quantecon/tests/test_inequality.py

```diff
@@ -0,0 +1,66 @@
+
+"""
+Tests for inequality.py
+
+"""
+
+import numpy as np
+from numpy.testing import assert_allclose
+from quantecon import lorenz_curve, gini_coefficient
+
+
+def test_lorenz_curve():
+    """
+    Tests `lorenz` function, which calculates the lorenz curve
+
+    An income distribution where everyone has almost the same wealth should
+    be similar to a straight line
+
+    An income distribution where one person has almost the wealth should
+    be flat and then shoot straight up when it approaches one
+    """
+    n = 3000
+
+    # Almost Equal distribution
+    y = np.repeat(1, n) + np.random.normal(scale=0.0001, size=n)
+    cum_people, cum_income = lorenz_curve(y)
+    assert_allclose(cum_people, cum_income, rtol=1e-03)
+
+    # Very uneven distribution
+    y = np.repeat(0.001, n)
+    y[4] = 100000
+    pop_cum, income_cum = lorenz_curve(y)
+    expected_income_cum = np.repeat(0., n + 1)
+    expected_income_cum[-1] = 1.
+    assert_allclose(expected_income_cum, income_cum, atol=1e-4)
+
+
+def test_gini_coeff():
+    """
+    Tests how the `Gini` object calculates the Gini coefficient
```

Comment:
Small note, since you now have functions rather than a class (to allow for jitting), perhaps the documentation should refer to the function, not the object

Reply:
Yep! I forgot to change that. Will change that too